backend/tests/modules/user/antisybil/test_antisybil.py

```diff
@@ -43,6 +43,41 @@ def test_antisybil_service(
     assert score == 2.572
 
 
+def test_guest_stamp(
+    patch_gitcoin_passport_issue_address_for_scoring,
+    patch_gitcoin_passport_fetch_score,
+    patch_gitcoin_passport_fetch_stamps,
+    mock_users_db,
+):
+    context = get_context(4)
+
+    service = GitcoinPassportAntisybil()
+    alice, _, _ = mock_users_db
+
+    score, expires_at, stamps = service.fetch_antisybil_status(context, alice.address)
```

Comment:
why we call fetch_antisybil_status here if it's called in update_antisybil_status?